src/systems/breadcrumbs/Breadcrumbs.cc

```diff
@@ -385,6 +400,30 @@ void Breadcrumbs::OnDeploy(const msgs::Empty &)
   IGN_PROFILE("Breadcrumbs::PreUpdate");
   {
     std::lock_guard<std::mutex> lock(this->pendingCmdsMutex);
+
+    // Check topic statistics for dropped messages
+    if (this->topicStatistics)
+    {
+      ignmsg << "Received breadcrumb deployment for " <<
+        this->modelRoot.ModelByIndex(0)->Name() << std::endl;
+      std::optional<transport::TopicStatistics> stats =
+        this->node.TopicStats(this->topic);
+      if (stats)
+      {
+        if (stats->DroppedMsgCount() > 0)
+        {
+        ignwarn << "Dropped message count of " << stats->DroppedMsgCount()
```

Comment:
nit: indentation

Reply:
Fixed in 8f861132968765420150dfa6d7e165c7aa4b7dc4